cnf-tests/testsuites/e2esuite/vrf/vrf_sriov.go

```diff
@@ -6,6 +6,8 @@ import (
 	"fmt"
 	"time"
 
+	"github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/pkg/networks"
```

Comment:
please move the import down to the cnf-tests section

Reply:
removed, as it wasn't actually used